lib/ACL/Rule.php

```diff
@@ -98,7 +98,7 @@ public function xmlSerialize(Writer $writer) {
 		$writer->write($data);
 	}
 
-	public function jsonSerialize() {
+	public function jsonSerialize(): array {
```

Comment:
```suggestion
	#[\ReturnTypeWillChange]
	public function jsonSerialize() {
```

So that it is still compatible with older version